core/aggsigdb/memory_v2.go

```diff
@@ -4,7 +4,6 @@ package aggsigdb
 
```

Comment:
TBH we can revisit this memorydbV2.
Now that we have metrics of the types of feature flags operators have turned on, we can see how many have this V2 DB. I personally do not have the perspective how much it improves Charon.
I can do it as part of looking into the aggregation failures we see recently, as they are related to the DB.